app/repositories/SessionCacheRepository.scala

```diff
@@ -89,9 +89,13 @@ class DefaultSessionCacheRepository @Inject()(mongoComponent: MongoComponent,
 
```

Comment:
At line no 67, link.accountNumber.size could be replaced with link.accountNumber.length. If this is to be addressed in the further PRs then ignore it.